Tasks/NuGetCommand/Common/VstsNuGetCommandHelper.ts

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+export function logFileNotFoundWarning(tl, filesList, searchPatternArray): void {
```

Comment:
This fix should go in to NuGetCommand and DotNetCoreCLI, and therefore this common logging function should be in tasks/nuget-task-common